lib/src/workspace.rs

```diff
@@ -580,6 +573,18 @@ impl DefaultWorkspaceLoader {
             working_copy_state_path,
         })
     }
+
+    fn load_working_copy(
+        &self,
+        store: &Arc<Store>,
+        working_copy_factory: &dyn WorkingCopyFactory,
+    ) -> Result<Box<dyn WorkingCopy>, WorkspaceLoadError> {
+        Ok(working_copy_factory.load_working_copy(
+            store.clone(),
+            self.workspace_root.clone(),
+            self.working_copy_state_path.clone(),
+        )?)
```

Comment:
nit: or just inline it because this is merely a forwarding function?

Reply:
True, that's even better. Done.